src/lib/Components/__tests__/FilterModal-tests.tsx

```diff
@@ -1,22 +1,82 @@
-import { Theme } from "@artsy/palette"
-import { FilterModal } from "lib/Components/FilterModal"
+import { Sans, Theme } from "@artsy/palette"
+import { SortOptionsScreen } from "lib/Components/ArtworkFilterOptions/SortOptions"
+import { FakeNavigator as MockNavigator } from "lib/Components/Bidding/__tests__/Helpers/FakeNavigator"
 import React from "react"
-import "react-native"
-import * as renderer from "react-test-renderer"
+import { act, create } from "react-test-renderer"
+import { FilterHeader, FilterOptions } from "../FilterModal"
 
-it("renders a snapshot of the filter artworks modal", () => {
-  let props
+let mockNavigator: MockNavigator
 
-  props = {
-    visible: true,
-  }
+beforeEach(() => {
+  mockNavigator = new MockNavigator()
+})
+
+describe("Filter options screen", () => {
+  it("renders the Filter Options screen properly", () => {
+    const FilterOptionsScreen = create(
+      <Theme>
+        <FilterOptions navigator={mockNavigator as any} closeModal={jest.fn()} />
+      </Theme>
+    )
+    expect(FilterOptionsScreen).toMatchSnapshot()
```

Comment:
Here too, I think this test would be more meaningful with more explicit assertions! (But not a blocker)